### PR TITLE
v0.7-alpha updates

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

File: spiceaidocs/content/en/reference/Spicepod/_index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - from: models/gas_fees
+```
+
+## `dependencies`
+
+A list of dependent Spicepods.
+
+```yaml
+dependencies:
+  - lukekim/demo:0.1
```
Comment (ewgenius) [inline spiceaidocs/content/en/reference/Spicepod/_index.md]: @phillipleblanc pointed out, that we should also allow to specify separate components of a spicepod to depend on

File: spiceaidocs/content/en/reference/Spicepod/_index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+functions:
+  - from: functions/nft_mint_counter
+```
+
+A function with a dependency on a dataset.
+
+```yaml
+functions:
+  - from: functions/nft_mint_counter
+    dependsOn: datasets/uniswap_v2_eth_usdc
```
Comment (ewgenius) [inline spiceaidocs/content/en/reference/Spicepod/_index.md]: @lukekim I think `dependsOn` should be an array

File: spiceaidocs/content/en/reference/Spicepod/_index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+## `dependencies`
+
+A list of dependent Spicepods.
+
+```yaml
+dependencies:
+  - lukekim/demo:0.1
+  - spicehq/nfts
```
Comment (ewgenius) [inline spiceaidocs/content/en/reference/Spicepod/_index.md:95]: - `spicehq/nfts` - latest production from github (iteration 1)
spice app has to be public (= it has connected public repository)

later (TBD):
- `spicehq/nfts@<sha>` - commit sha
- `spicehq/nfts@0.0.1` - Published spicepod in spicerack

File: spiceaidocs/content/en/reference/Spicepod/_index.md (hunk 2 above)
Comment (ewgenius) [reply spiceaidocs/content/en/reference/Spicepod/_index.md]: TODO: decide on casing

**Review by ewgenius — APPROVED** [review]

**Review by digadeesh — APPROVED** [review]

**Pull request merged**